packages/loader/src/ktx2/KTX2Loader.ts

```diff
@@ -40,12 +37,14 @@ export class KTX2Loader extends Loader<Texture2D | TextureCube> {
       KTX2TargetFormat.ETC,
       KTX2TargetFormat.BC1_BC3,
       KTX2TargetFormat.PVRTC
-    ]
+    ],
+    hdr: [KTX2TargetFormat.BC6H, KTX2TargetFormat.RGBA16]
```

Comment:
what should KTX2Loader do if `textureHalfFloat` is not supported in WebGL1?

Reply:
I think we should throw an error.